ServerCore/Pages/Responses/CreateBulk.cshtml.cs

```diff
@@ -53,6 +54,8 @@ public async Task<IActionResult> OnPostAsync(int puzzleId)
             StringReader responseTextReader = new StringReader(ResponseText ?? string.Empty);
             StringReader noteReader = new StringReader(Note ?? string.Empty);
 
+            HashSet<string> submissions = new HashSet<string>();
```

Comment:
unused? Do you need to build this and use it as well as checking existing responses?

Reply:
It's used on line 109 to build and check.